### ISSUE TITLE
How to apply Windows path to videoclip.conf?

### ISSUE THREAD

**Issue by knuurr** [Ajatt-Tools/videoclip#21]
I'm having troubles with WIndows paths. For example:

```
video_folder_path=C:\Users\qbus\Pictures
```
This path when pasted into Explorer displays fine

![obraz](https://github.com/Ajatt-Tools/videoclip/assets/135069967/f568fa17-579b-4b9c-b6a9-136cded8fbd5)

But when used with script, it's reported that file can't be created. Tested on latest version.

![obraz](https://github.com/Ajatt-Tools/videoclip/assets/135069967/484a9c98-e2d1-4f46-8a49-6ee74524cf22)

**Comment by tatsumoto-ren** [comment]
[Here is a list of recommended GNU distributions](https://tatsumoto.neocities.org/blog/our-immersion-learning-toolset#gnu-linux).

**Comment by knuurr** [comment]
Yeah that's funny, but you know since you provide WIndows installation and paths I hoped it actually works on Windows.

**Comment by tatsumoto-ren** [comment]
> I hoped it actually works on Windows.

Does it work with the default config file? If not, the problem isn't actually in the way you specify paths.
